packages/client/wallets/aa/src/blockchain/wallets/passkey.ts

```diff
@@ -0,0 +1,55 @@
+import { CrossmintWalletService, EVMAAWallet, PasskeyWalletConfig } from "@/index";
+import { getIdString } from "@/utils/user";
+import { createPasskeyValidator, deserializePasskeyValidator } from "@zerodev/passkey-validator";
+import { KernelValidator, createKernelAccount } from "@zerodev/sdk";
+import { ENTRYPOINT_ADDRESS_V07 } from "permissionless";
+import { ENTRYPOINT_ADDRESS_V07_TYPE } from "permissionless/_types/types";
+import { HttpTransport, PublicClient } from "viem";
+
+import { EVMBlockchainIncludingTestnet, UserIdentifier } from "@crossmint/common-sdk-base";
+
+export default class PasskeyWalletService {
+    private readonly entryPoint = ENTRYPOINT_ADDRESS_V07;
```

Comment:
We probably need to add the entry point verification. Apart from backwards compatibility. some chains like astar don't support v07 and only work for v06